include/OpenColorIO/OpenColorIO.h

```diff
@@ -3054,6 +3054,10 @@ class OCIOEXPORT GpuShaderCreator
     virtual void setTextureMaxWidth(unsigned maxWidth) = 0;
     virtual unsigned getTextureMaxWidth() const noexcept = 0;
 
+    /// Allow 1D GPU resource type, otherwise always using 2D resources for 1D LUTs.
```

Comment:
Do we still need these functions with the new `dimensions` parameter in the `add/getTexture` interface?